### DISCUSSION TITLE
Issue creating renormalized HA FCs

### DISCUSSION THREAD

**Discussion by fatimah-ma** [ttadano/alamode#19] [Q&A] [answered]
Hello!

I am trying to extract the new, renormalized harmonic FCs after the SCPH run using dfc2, but I keeping running into this error: 

Warning: The following force constant doesn't exist in the original file:
    0    1    1    8    1    0    2   -1.08232e-10

and it goes on and on.
I checked the output for the original harmonic forces and the new anharmonic forces, and both ran without error. I am not sure where this error could be coming from or how to fix it, could you please guide me through that? 

Thank you!

**Comment by ttadano** [comment, marked as answer]
Hi,

You can neglect this warning as long as the correction term (the last column) is small.

This warning is raised when the absolute value of the correction is larger than 1.0e-10 and the corresponding force constant does not exist in the original FCSXML file. In your case, the correction term is only slightly above the tolerance. So, it should not cause any problems hopefully.

When the correction term is much larger than 1.0e-10, this warning indicates a possible error in the inputs. Likely errors are
- the input structure is not fully symmetrized, or the numerical digits of lattice vectors and fractional coordinates are insufficient.
- the given ``KMESH_INTERPOLATE`` is not commensurate with the supercell size of the original FCSXML.
